<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optDialog</class>
 <widget class="QDialog" name="optDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget_menu">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_mappear">
      <attribute name="icon">
       <iconset resource="../gpxsee.qrc">
        <normaloff>:/icons/preferences-desktop-display.png</normaloff>:/icons/preferences-desktop-display.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget_appear">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_color">
          <attribute name="title">
           <string>Colors</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="margin">
              <number>3</number>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="label_bcolor">
               <property name="text">
                <string>Base color:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_pshift">
               <property name="text">
                <string>Palette shift:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_color_pshift"/>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="pushButton_color_base">
                 <property name="text">
                  <string>Color</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_path">
          <attribute name="title">
           <string>Path</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>593</width>
                <height>408</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="margin">
                  <number>4</number>
                 </property>
                 <item>
                  <layout class="QFormLayout" name="formLayout_2">
                   <property name="fieldGrowthPolicy">
                    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                   </property>
                   <property name="margin">
                    <number>4</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Track width:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QSpinBox" name="spinBox_path_trwidth"/>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_3">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Route width:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QSpinBox" name="spinBox_path_rwidth"/>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_4">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item row="6" column="0">
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>Route style:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="1">
                    <widget class="StyleComboBox" name="comboBox_path_rstyle"/>
                   </item>
                   <item row="1" column="1">
                    <widget class="StyleComboBox" name="comboBox_path_trstyle"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Track style:</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_path_antia">
                   <property name="text">
                    <string>Use anti-aliasing</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_graph">
          <attribute name="title">
           <string>Graphs</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <property name="margin">
            <number>3</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="margin">
              <number>4</number>
             </property>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QSpinBox" name="spinBox_graph_lwidth"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Line width:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="margin">
              <number>4</number>
             </property>
             <item>
              <widget class="QCheckBox" name="checkBox_graph_antia">
               <property name="text">
                <string>Use anti-aliasing</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_mpoi">
      <attribute name="icon">
       <iconset resource="../gpxsee.qrc">
        <normaloff>:/icons/flag_48.png</normaloff>:/icons/flag_48.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget_poi">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_poi">
          <attribute name="title">
           <string>Poi</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <property name="margin">
            <number>4</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_poi_radius"/>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_poir">
               <property name="text">
                <string>POI radius:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_mexport">
      <attribute name="icon">
       <iconset resource="../gpxsee.qrc">
        <normaloff>:/icons/document-print-preview.png</normaloff>:/icons/document-print-preview.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget_export">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_header">
          <attribute name="title">
           <string>Header</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QCheckBox" name="checkBox_exp_name">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_exp_date">
             <property name="text">
              <string>Date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_exp_distance">
             <property name="text">
              <string>Distance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_exp_time">
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_exp_movingtime">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Moving Time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_exp_itemcount">
             <property name="text">
              <string>Item count (&gt;1)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_egraph">
          <attribute name="title">
           <string>Graphs</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QCheckBox" name="checkBox_exp_separate">
             <property name="text">
              <string>Separate graph page</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_msystem">
      <attribute name="icon">
       <iconset resource="../gpxsee.qrc">
        <normaloff>:/icons/system-run.png</normaloff>:/icons/system-run.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget_system">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_system">
          <attribute name="title">
           <string>System</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <widget class="QCheckBox" name="checkBox_system_opengl">
             <property name="text">
              <string>Use OpenGl</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyleComboBox</class>
   <extends>QComboBox</extends>
   <header>src/stylecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gpxsee.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>optDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>optDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
